getDayName.js

```diff
@@ -0,0 +1,7 @@
+// The days of the week are: "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"
+function getDayName(dateString) {
```

Comment:
what is the type of argument. specify type or add some checks because now we can also pass undefined in this.